lib/standard/text/ropes.nit

```diff
@@ -82,6 +82,12 @@ private class Concat
 
 	redef fun empty do return ""
 
+	# Cache for the latest accessed FlatString in `self`
+	var flat_cache: FlatString is noinit
+
+	# Position of the beginning of `flat_cache` in `self`
+	var flat_last_pos_start: Int = -1
```

Comment:
Related to my previous comment: here you use a default value since cache has not been accessed. You should follow the same principle for `flat_cache`.